bundles/org.openhab.binding.e3dc/README.md

```diff
@@ -0,0 +1,371 @@
+| Wallbox Available     | wb-available    |  Switch | RO | Indicates if the Wallbox is attached. Check your Wallbox ID in offline case  |
+| Sun Mode              | wb-sunmode |  Switch | RW  | Activate / Deactivate Sun Mode. Off case takes Grid Power to ensure highest possible charging.   |
+| Wallbox Charging Aborted | wb-charging-aborted |  Switch | RW  | Indicates if Wallbox charging is aborted  |
+| Wallbox Charging      | wb-charging |  Switch | RO  | Indicates your Wallbox is charging   |
+| Jack Locked           | wb-jack-locked  |  Switch | RO  | Indicates your Jack is locked   |
+| Jack Plugged          | wb-jack-plugged |  Switch | RO  | Indicates your Jack is plugged    |
+| Schuko Socket On      | wb-schuko-on    |  Switch  | RW | If your Wallbox has an additional Schuko Socket it provides state ON or OFF    |
+| Schuko Socket Plugged | wb-schuko-plugged |  Switch | RO  |If your Wallbox has an additional Schuko Socket it provides plugged state ON or OFF    |
+| Schuko Socket Locked  | wb-schuko-locked |  Switch | RO  | If your Wallbox has an additional Schuko Socket it provides locked state ON or OFF |
+| Schuko 16A Relay On   | wb-schuko-relay-16a |  Switch | RO  | Indicates if Schuko 16A Relay is ON     |
+| 16A Relay On          | wb-relay-16a    |  Switch | RO  | Indicates if 16A Relay is ON     |
+| 32A Relay On          | wb-relay-32a    |  Switch | RO  | Indicates if 32A Relay is ON    |
+| 1-Phase Charging      | 1-Phase Active  |  Switch | RW  | Indicates if 1-phase charging is activated. If OFF 3-phase charging is activated    |
+
+### E3DC String Details Block
+
+| Channel Label         | Channel ID      | Type           | Description                  |
+|-----------------------|-----------------|----------------|------------------------------|
+| String 1 Potential    | string1-dc-voltage |  Number:Volt  | Volt on String 1           |
+| String 2 Potential    | string2-dc-voltage |  Number:Volt  | Volt on String 2           |
+| String 3 Potential    | string3-dc-voltage |  Number:Volt  | Volt on String 3           |
+| String 1 Current      | string1-dc-current |  Number:Ampere  | Ampere on String 1       |
+| String 2 Current      | string2-dc-current |  Number:Ampere  | Ampere on String 2       |
+| String 3 Current      | string3-dc-current |  Number:Ampere  | Ampere on String 3       |
+| String 1 Power        | string1-dc-output  |  Number:Power   | Watt produced by String 1 |
+| String 2 Power        | string2-dc-output  |  Number:Power   | Watt produced by String 2 |
+| String 3 Power        | string3-dc-output  |  Number:Power   | Watt produced by String 3 |
+
+
+### E3DC EMS Block
+
+| Channel Label         | Channel ID      | Type           | Description                  |
+|-----------------------|-----------------|----------------|------------------------------|
+| Emergency Power Status| emergency-power-status |  String  | Indicates if Emergency Power Supply is possible or not, active or inactive |
```

Comment:
You could add the values, the user can expect here.

Reply:
Added all possible EP Status values